src/App.tsx

```diff
@@ -163,11 +163,15 @@ export default function App() {
     const playerScore = score(hand).value;
     if (playerScore > dealerScore) {
       setMessage(Message.playerWin);
-    } else if (dealerScore > playerScore) {
+    } else if (dealerScore > playerScore && dealerScore <=21) {
       setMessage(Message.dealerWin);
-    } else {
+    } else if (dealerScore <=21 && dealerScore == playerScore) {
       setMessage(Message.push);
     }
+    else
```

Comment:
`ℹ️ nitpick:` Please put `else` on the same line as the closing and opening curly braces.